### DISCUSSION TITLE
New firmware for RLC-510A (V2)

### DISCUSSION THREAD

**Discussion by fjmartinez2k** [AT0myks/reolink-fw-archive#18] [Firmware sharing]
This is an updated firmware for the RLC-510A (hardware version 2)

https://support-d.reolink.com/attachments/token/axfGH42kwt74trKrKFysXe2Mg/?name=IPC_523128M5MP_V2.2109_23051501.RLC-510A.OV05A10.5MP.REOLINK.pak

Fixes live video delayed in SSS.

**Comment by AT0myks** [comment]
Thanks, what's SSS?

**Reply by fjmartinez2k** [reply]
SSS in my post is short for Sinology Surveillance Station

**Reply by AT0myks** [reply]
Added in ce45de4ec1af9ad15f4f523f4b39b40ebf4e479c.

**Comment by trox88** [comment]
Till this FW update, Vehicle detection always on parking cars, continuously getting notifications about parking cars. Had to decrease sensity to 0 to stop it.
Anyone has same problame?
